Otherwise Linux configs which are symlinks are copied as symlinks and prevent
build from going further afterwards.

Signed-off-by: Julien Boibessot <julien.boibessot@free.fr>

Index: buildroot/target/linux/Makefile.in.advanced
===================================================================
--- buildroot.orig/target/linux/Makefile.in.advanced	2010-10-28 16:51:17.000000000 +0200
+++ buildroot/target/linux/Makefile.in.advanced	2010-10-28 16:55:09.000000000 +0200
@@ -315,12 +315,12 @@
 ifeq ($(BR2_PACKAGE_LINUX_USE_KCONFIG),y)
 	# Try user defined config
 	if [ -f "$(LINUX26_KCONFIG)" ] ; then \
-		cp -dpf $(LINUX26_KCONFIG) $@ ; \
+		cp -pf $(LINUX26_KCONFIG) $@ ; \
 	fi
 	# Try to get a config for this linux version in the board directory
 	if [ ! -f "$(LINUX26_DIR)/.config" ] ; then \
 	    if [ -f "$(BOARD_PATH)/$(BOARD_NAME)-linux-$(LINUX26_VERSION).config" ] ; then \
-		cp -dpf $(BOARD_PATH)/$(BOARD_NAME)-linux-$(LINUX26_VERSION).config $@ ; \
+		cp -pf $(BOARD_PATH)/$(BOARD_NAME)-linux-$(LINUX26_VERSION).config $@ ; \
 	    fi ; \
 	fi
 	# Try to get a config for latest linux version in the board directory
